package_metrics/metrics.py

```diff
@@ -0,0 +1,48 @@
+import argparse
+
+from package_metrics.package_managers.pip import parse_pip
+
+
+def get_packages(package_manager):
+    parsers = {
+        "pip": parse_pip,
+    }
+    return parsers[package_manager]()
+
+
+def build_packages_table(packages):
+    def build_row(behind, name, current, latest):
+        return f"{behind:8s} {name:28s} {current:12s} {latest}"
+
+    rows = [build_row("Behind", "Package", "Latest", "Version")]
+
+    records = sorted(packages)
```

Comment:
How do you feel about having the option to sort the table based on how outdated the packages are?

Reply:
As it stands now, this sorts by the first value which is the "delta" or "behind" value, so it does sort by how outdated packages are, but puts the most out of date at the bottom, like so:
```
Behind   Package                      Latest       Version
0.1.0    packaging                    23.1         23.0
0.1.0    pip                          23.1         23.0.1
0.1.0    pip-tools                    6.13.0       6.12.3
0.2.0    wheel                        0.40.0       0.38.4
9.0.0    setuptools                   67.6.1       58.1.0
```
Do you think we should reverse the sort order?

Reply:
Yep. This should be fine. 
Thanks for providing this info.